README.md

```diff
@@ -2,26 +2,13 @@
 
 ### Install a local cluster
 
-#### Docker desktop
-
-Docker desktop includes the ability to run a local kubernetes cluster.
-
-The easiest way of doing this is to go to `Docker desktop -> Preferences (the little cog in top right) -> Kubernetes -> Select Enable Kubernetes`
-
-![docker desktop k8s](docker-desktop-k8s.png)
-
-*Note this will take a few minutes depending on your connection*
-
-We can then switch to this cluster using the Docker desktop icon in the tool bar at the top
-
-![docker desktop toolbar](docker-desktop-switch-context.png)
-
-
-<details>
-<summary><i> minikube alternative</i></summary>
 If Docker desktop is not an option, we can also use something called <code>minikube</code>
```

Comment:
Might be worth moving this line down to the Docker desktop alternative section?

Reply:
I've rewritten this a bit and moved it to the alt section